### PR TITLE
Lv-160

### PR DESCRIPTION
**Pull request by TamaraKovalchuk**
Fix UserDashboard.js and EditProfile.js. Gave the opportunity to change nickname. Check this. please.

### CODE REVIEW

File: src/modules/user-page/components/user-dashboard/UserDashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                      to={`/edit-profile/${this.props.userprofile.nickname}`}
+                    >
+                      Edit profile
+                    </Link>
+                  </Button>
+                : ''}
+            </Grid>
+            <Grid className="all-infa" container>
+              <div><span>
+                {this.props.userprofile.posts ? this.props.userprofile.posts.length: "0"}
```
Comment (Jestermaxrko) [inline src/modules/user-page/components/user-dashboard/UserDashboard.jsx]: use ' instead of "

File: src/modules/_index.scss [hunk 2]
```diff
@@ -12,9 +12,9 @@
...
 @import './email-verify/index';
 
-@import './user-page/components/dashboard/index';
+@import './user-page/components/user-dashboard/index';
 @import './user-page/components/user-posts/index';
 @import './user-page/components/post-page/index';
 @import './user-page/components/user-image/index';
-@import './user-page/components/dashboard/index';
+// @import './user-page/components/user-dashboard/index';
 @import './edit-profile/index';
```
Comment (Jestermaxrko) [inline src/modules/_index.scss]: add new line in the end

File: src/modules/user-page/components/user-dashboard/UserDashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+              </span> posts</div>
+              <div className="followers"><span>
```
Comment (Jestermaxrko) [inline src/modules/user-page/components/user-dashboard/UserDashboard.jsx]: add a class to spans like this

File: src/reducers/userprofile.js [hunk 4]
```diff
@@ -12,13 +12,6 @@ const initialState = {
...
-        case 'USER_PROFILE_EDIT':
-        return {
-            ...state,
-            error: false,
-            errorMessage: null,
-            userprofile: action.payload
-        };
         case 'USER_PROFILE_UPDATED':
         return {
             ...state,
```
Comment (Jestermaxrko) [inline src/reducers/userprofile.js]: dont copy all state if you rewrite every field of it

File: src/modules/user-page/components/user-dashboard/UserDashboard.jsx (hunk 1 above)
Comment (Jestermaxrko) [inline src/modules/user-page/components/user-dashboard/UserDashboard.jsx]: use ' instead of "

File: src/modules/_index.scss (hunk 2 above)
Comment (Jestermaxrko) [inline src/modules/_index.scss]: add new line in the end

File: src/modules/user-page/components/user-dashboard/UserDashboard.jsx (hunk 3 above)
Comment (TamaraKovalchuk) [reply src/modules/user-page/components/user-dashboard/UserDashboard.jsx]: <div className="quantity"><span className="number">

Comment (TamaraKovalchuk) [reply src/modules/user-page/components/user-dashboard/UserDashboard.jsx]: <div className="quantity"><span className="number">

File: src/reducers/userprofile.js (hunk 4 above)
Comment (TamaraKovalchuk) [reply src/reducers/userprofile.js]: ok

File: src/modules/_index.scss (hunk 2 above)
Comment (TamaraKovalchuk) [reply src/modules/_index.scss]: оk

File: src/modules/user-page/components/user-dashboard/UserDashboard.jsx (hunk 1 above)
Comment (TamaraKovalchuk) [reply src/modules/user-page/components/user-dashboard/UserDashboard.jsx]: ok

**Pull request closed without merging**